q2_types/feature_data/_transformer.py

```diff
@@ -359,3 +359,18 @@ def _19(data: AlignedDNAIterator) -> AlignedDNAFASTAFormat:
 @plugin.register_transformer
 def _33(ff: AlignedDNAFASTAFormat) -> qiime2.Metadata:
     return _dnafastaformats_to_metadata(ff)
+
+
+# differential types
+@plugin.register_transformer
```

Comment:
If my comment above re Metadata is applicable, this transformer could be re-implemented as shown [here](https://github.com/qiime2/q2-demux/blob/master/q2_demux/_transformer.py#L127-L129).